src/main/java/frc/robot/HwMotor.java

```diff
@@ -64,29 +64,6 @@
 
     NetworkTableInstance inst = NetworkTableInstance.getDefault();
     NetworkTable table = inst.getTable("datatable");
-    private NetworkTableEntry pre_kP = table.getEntry("shooter_kP");
-    private NetworkTableEntry pre_kF = table.getEntry("shooter_kF");
-    private NetworkTableEntry pre_kI = table.getEntry("shooter_kI");
-    private NetworkTableEntry pre_kD = table.getEntry("shooter_kD");
-    private NetworkTableEntry intake_pre_kP = table.getEntry("intake_kP");
-    private NetworkTableEntry intake_pre_kF = table.getEntry("intake_kF");
-    private NetworkTableEntry intake_pre_kI = table.getEntry("intake_kI");
-    private NetworkTableEntry intake_pre_kD = table.getEntry("intake_kD");
-    public NetworkTableEntry shooterVel = table.getEntry("ShooterVelocity");
-    public NetworkTableEntry intakeVel = table.getEntry("IntakeVelocity");
-    public NetworkTableEntry intakeTemp = table.getEntry("IntakeTemp");
-    public NetworkTableEntry shooterOutput = table.getEntry("ShooterOutput");
-    public NetworkTableEntry shooterVelSetPoint = table.getEntry("ShooterVelocitySetPoint");
-    private NetworkTableEntry drivePrekP = table.getEntry("drivekP");
-    private NetworkTableEntry drivePrekI = table.getEntry("drivekI");
-    private NetworkTableEntry drivePrekD = table.getEntry("drivekD");
-    private NetworkTableEntry drivePrekF = table.getEntry("drivekF");
-    public NetworkTableEntry rightDriveVel = table.getEntry("rightDriveVelocity");
-    public NetworkTableEntry rightDriveOutput = table.getEntry("rightDriveOutput");
-    public NetworkTableEntry rightDriveVelSetpoint = table.getEntry("rightSetpointReadout");
-    public NetworkTableEntry leftDriveVel = table.getEntry("leftDriveVelocity");
-    public NetworkTableEntry leftDriveOutput = table.getEntry("leftDriveOutput");
-    public NetworkTableEntry leftDriveVelSetpoint = table.getEntry("leftSetpointReadout");
     public NetworkTableEntry ball1 = table.getEntry("BallAStatus");
```

Comment:
You missed getting rid of these NetworkTableEntry after you moved them into Table.java